File: server-scala-play-server/app/model/TimeSlotResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for TimeSlotResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class TimeSlotResponse(
  scheduledDate: Option[Long],
  startTime: Option[Long],
  endTime: Option[Long],
  dayOfWeek: Option[Int],
  timeZone: Option[String],
  reservationCount: Option[Int],
  reservations: Option[List[AccountShortResponse]]
)

object TimeSlotResponse {
  implicit lazy val timeSlotResponseJsonFormat: Format[TimeSlotResponse] = Json.format[TimeSlotResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/TimeSlotResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class TimeSlotResponse 
{
    public long ScheduledDate { get; set; }
    public long StartTime { get; set; }
    public long EndTime { get; set; }
    public int DayOfWeek { get; set; }
    public string VarTimeZone { get; set; }
    public int ReservationCount { get; set; }
    public List<AccountShortResponse> Reservations { get; set; }
}


